cypress/scripts/linux-brave-version.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+BRAVE_PATH=$(which brave-browser)
```

Comment:
`/usr/bin/which` might be a safer way to use `which`, since `which` is often aliased by some linux systems with a few additional flags like `--show-dot` or `--show-tilde`. Not sure how cypress handles these formats.

Reply:
Nice catch - @rbeer as a Linux user, would you have a moment to test that @gomesalexandre's suggested change still works on your machine?

Reply:
As discussed with @gomesalexandre via DM - this is an edge case.

So as not to block this from being merged, I've added a TODO comment, that can be tested by someone with Linux, in https://github.com/shapeshift/web/pull/954/commits/3496da207ee3b35c5dc0061e6d9d6c51c6074ee8.